test/02LessonTwoTest.js

```diff
@@ -7,11 +7,32 @@ function whichSpecies(character) {
     // should return "cat" when character is 'garfield'
     // should return "fish" when character is 'nemo'
     // should return false if character is anything else
+    
+    if (character === 'scooby'){
+        return "dog";
+    }
+
+    if (character === 'garfield'){
+        return "cat";
+    }
+
+    if (character === 'nemo'){
+        return "fish";
+    }
+    
+    else {return false;}
+
+
 
 }
 
 function isEven(number) {
     // should return true is number is even (divisible by 2)
+    if ( (number % 2) === 0){
+        return true;
+
+    }
+        
 
```

Comment:
I see you're making progress catching up!

You're missing something here, though. **isEven(number)** should return false when the number is not even.